README.md

```diff
@@ -101,8 +101,17 @@ cp -R dagster.yaml ~/dagster_home/
 dagit -w workspace.yaml
 ```
 
-After you have finished the above steps, you can open your localhost or http://127.0.0.1:3000 and you will be able to see in the launcher the list of tests or collections available.
+After you have finished the above steps, you can open your localhost or http://127.0.0.1:3000 and you will be able to see in the launcher the list of tests or collections available. You can click on the Launchpad tab.
 
+![api to amt dagster job](docs\images\pipe_api_job.jpg)
```

Comment:
The preview is not loading because of the image path. It should be forward slash 
https://gist.github.com/TT--/14260aef6c0e31fca5b37e7cb3c53020

https://github.com/Ed-Fi-Exchange-OSS/API-to-AMT-Data-Lake/tree/BIA-1344
![image](https://user-images.githubusercontent.com/5459528/231816866-459613d8-6007-4645-8c54-d56bbfc7fdba.png)